data-processing/common/commands/fetch_s2_data.py

```diff
@@ -59,7 +59,6 @@ def _mk_api_request(self, arxivId: ArxivId) -> requests.Response:
         headers = None
 
         if self._partner_api_token:
-            base_url = "partner.semanticscholar.org"
```

Comment:
Don't we need to replace this with something?

Reply:
no, it'll just stay as the regular "api.semanticscholar.org" on L58.